iFixit/react-commerce
# Refactor footer

fixes #230 
 
## Changelog

- Move footer components into `react-commerce` monorepo
- Refactor newsletter section according to the new design

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com) 
2. Visit the parts product list
3. Verify that the newsletter section is implemented according to the [new design](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit-Shopify-Working-Files?node-id=3577%3A29376)
